src/Blocks/BlockHandler.cpp

```diff
@@ -656,6 +656,15 @@ bool cBlockHandler::ToolHasSilkTouch(const cItem * a_Tool)
 
 
 
+unsigned int cBlockHandler::ToolFortuneLevel(const cItem * a_Tool)
+{
+	return (a_Tool != nullptr)*a_Tool->m_Enchantments.GetLevel(cEnchantments::enchFortune);
+}
```

Comment:
Please, no! What's wrong with a normal if statement? We're using C++, not playing golf with C...